lib/constants/enums.ts

```diff
@@ -14,22 +14,50 @@ export enum OrderSide {
   Sell,
 }
 
-export enum Network {
+export enum XUNetwork {
+  // real coins.
   MainNet = 'mainnet',
+
+  // pre-defined testnet per currency (e.g. bitcoin: testnet3, litecoin: testnet4).
   TestNet = 'testnet',
+
+  // coins are minted by Exchange Union.
   SimNet = 'simnet',
+
+  // coins are minted privately. Seed nodes are not configurable.
   RegTest = 'regtest',
 }
 
 /**
- * Magic value indicating wire msg origin network, and used to seek to next msg when stream state is unknown
+ * LN network config per XU network.
  */
-export enum NetworkMagic {
-  MainNet = 0xd9b4bef9,
-  TestNet = 0x0709110b,
-  SimNet = 0x12141c16,
-  RegTest = 0xdab5bffa,
-}
+export const lnNetworks = {
+  [XUNetwork.MainNet]: 'mainnet',
+  [XUNetwork.TestNet]: 'testnet',
+  [XUNetwork.SimNet]: 'testnet',
+  [XUNetwork.RegTest]: 'testnet',
+};
+
+/**
+ * Magic value per XU network, indicating wire msg origin network,
+ * and used to seek to next msg when stream state is unknown.
+ */
+export const xuNetworkMagicVals = {
+  [XUNetwork.MainNet]: 0xd9b4bef9,
+  [XUNetwork.TestNet]: 0x0709110b,
+  [XUNetwork.SimNet]: 0x12141c16,
+  [XUNetwork.RegTest]: 0xdab5bffa,
+};
+
+/**
+ * XU network per magic value.
+ */
+export const magicValsXuNetwork: { [val: number]: XUNetwork} = {
```

Comment:
I added it because otherwise I'm getting a compilation error [here](https://github.com/ExchangeUnion/xud/blob/xunetworks/lib/p2p/Framer.ts#L129).

````
 TS7017: Element implicitly has an 'any' type because type '{ [0xd9b4bef9]: XUNetwork; [0x0709110b]: XUNetwork; [0x12141c16]: XUNetwork; [0xdab5bffa]: XUNetwork; }' has no index signature.
````

Reply:
Gotcha, I see that too.

Well one other approach - although the compiler still implicitly converts this to a `{ [val: number]: XUNetwork }` type - is referring to the magic values directly instead of copy and paste.

```javascript
export const magicValsXuNetwork = {
  [xuNetworkMagicVals[XUNetwork.MainNet]]: XUNetwork.MainNet,
  [xuNetworkMagicVals[XUNetwork.TestNet]]: XUNetwork.TestNet,
  [xuNetworkMagicVals[XUNetwork.SimNet]]: XUNetwork.SimNet,
  [xuNetworkMagicVals[XUNetwork.RegTest]]: XUNetwork.RegTest,
};
```

I'd prefer that slightly, although I'd be fine with it either way, so go with what you prefer.

Reply:
Looks good, will do.